test/e2e/util/util.go

```diff
@@ -139,7 +143,7 @@ func WaitForGuestClient(t *testing.T, ctx context.Context, client crclient.Clien
 
 	t.Logf("Waiting for a successful connection to the guest apiserver")
 	var guestClient crclient.Client
-	waitForGuestClientCtx, cancel := context.WithTimeout(ctx, 5*time.Minute)
+	waitForGuestClientCtx, cancel := context.WithTimeout(ctx, 15*time.Minute)
```

Comment:
why is this increasing?

Reply:
kubeconfig takes longer to report available after switching from private to public

Reply:
Please let's add a comment that justifies the change for future context.